cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -1641,6 +1608,14 @@ msgstr "Desempleado o despedido"
 msgid "Retired"
 msgstr "Jubilado"
 
+#, python-format
+#~ msgid ""
+#~ "Join the conversation. Follow CFPB on %(twitter_link)s and "
+#~ "%(facebook_link)s."
+#~ msgstr ""
+#~ "Únase a la conversación. Síganos en %(twitter_link)s y en "
+#~ "%(facebook_link)s."
+
```

Comment:
The lines in `blog_page.html` that contain this text have not changed in 7 months. Why would they be getting commented out here?